pandas/tests/extension/test_datetime.py

```diff
@@ -82,7 +82,15 @@ class TestDatetimeDtype(BaseDatetimeTests, base.BaseDtypeTests):
 
 
 class TestConstructors(BaseDatetimeTests, base.BaseConstructorsTests):
-    pass
+    @pytest.mark.xfail(reason="GH-26469")
+    def test_series_constructor_scalar_with_index(self, data):
+        # TypeError: data type not understood
+        super().test_series_constructor_scalar_with_index(data)
+
+    @pytest.mark.xfail(reason="GH-26469")
+    def test_series_constructor_scalar_with_one_element_index(self, data):
+        # TypeError: data type not understood
+        super().test_series_constructor_scalar_with_one_element_index(data)
```

Comment:
there is some special casing for datetime in the Series construction. Although fixing this could also be considered out-of-scope for the issue that this PR attempts to close, I could look into this further and maybe raise a separate issue if not fixed here.

Reply:
Opening a separate issue is fine for me as well, either way

Reply:
this is not failing with the changes as they stand atm. the previous 'fix' was inside a `if isinstance(dtype, ExtensionDtype) and dtype.kind != "M"` and hence not applicable to DatetimeTZDtype